### DISCUSSION TITLE
Remove header

### DISCUSSION THREAD

**Discussion by blasterkings** [dwainscheeren/dwains-lovelace-dashboard#773] [General]
Hi i would like to know if it is possible to remove this header. I don't use any icons other than home screen. 

![image](https://github.com/dwainscheeren/dwains-lovelace-dashboard/assets/61855554/c7137230-9f95-4165-ab03-fedc32a84354)

If it's possible - please give me instructions on script on file name to modify.

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
